src/components/Accounts/ChargeFeeFine/Charge.js

```diff
@@ -284,6 +289,65 @@ class Charge extends React.Component {
     this.callout.sendCallout({ message });
   }
 
+  showConfirmDialog = (values) => {
+    this.setState({
+      showConfirmDialog: true,
+      values
+    });
+
+    return new Promise((resolve, reject) => {
+      this.actionResolve = resolve;
+      this.actionReject = reject;
+    });
+  }
+
+  hideConfirmDialog = () => {
+    this.setState({
+      showConfirmDialog: false,
+      values: {}
+    });
+  }
+
+  onConfirm = () => {
+    const { values } = this.state;
+    this.onClickCharge(this.type).then(() => {
+      this.type.remaining = parseFloat(this.type.amount - values.amount).toFixed(2);
+      if (this.type.remaining === '0.00') {
+        this.type.paymentStatus.name = 'Paid Fully';
```

Comment:
`Paid Fully`, `Closed`, and `Paid Partially` all need to use `<FormattedMessage>`.